<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugPage</class>
 <widget class="QWizardPage" name="DebugPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>349</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="debug">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Continue debugging.</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="HighlightedListView" name="shortcuts">
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="shortcutsEditPanel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="newBtn">
        <property name="toolTip">
         <string>Create a new shortcut</string>
        </property>
        <property name="text">
         <string>Create Shortcut</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Edit the selected shortcut</string>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="shortcutsSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>310</width>
          <height>14</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="delBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Delete the selected shortcut</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="reinstall">
     <property name="text">
      <string>Select another solution(or edit the current solution).</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="end">
     <property name="text">
      <string>Finish debugging.</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HighlightedListView</class>
   <extends>QListView</extends>
   <header>src/highlightedlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>editBtn</sender>
   <signal>clicked()</signal>
   <receiver>DebugPage</receiver>
   <slot>edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>edit()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
